editor/plugins/node_3d_editor_gizmos.cpp

```diff
@@ -5968,8 +5968,8 @@ void FogVolumeGizmoPlugin::redraw(EditorNode3DGizmo *p_gizmo) {
 
 		Vector<Vector3> lines;
 		AABB aabb;
-		aabb.position = -cs->call("get_extents").operator Vector3();
-		aabb.size = aabb.position * -2;
+		aabb.size = -cs->call("get_size").operator Vector3();
+		aabb.position = aabb.size / 2;
```

Comment:
This seems mixed up. Its likely responsible for the broken gizmo in my screenshot
```suggestion
		aabb.size = cs->call("get_size").operator Vector3();
		aabb.position = -aabb.size / 2;
```

Reply:
Fixed